image/classpath

```diff
@@ -1,9 +1,24 @@
 mule-licence
 mule-app-properties
+mule-app-secrets
```

Comment:
This is a bit too opinionated for this repository.

Reply:
Not sure why, but sorted.